Cython/Utility/ObjectHandling.c

```diff
@@ -2685,22 +2685,42 @@ static CYTHON_INLINE int __Pyx_object_dict_version_matches(PyObject* obj, PY_UIN
 }
 #endif
 
+////////////// CachedMethodType.proto //////////////
+
+#if CYTHON_COMPILING_IN_LIMITED_API
+static PyObject *__Pyx_CachedMethodType = NULL;
```

Comment:
Should we clean this up on shutdown? I think there's a utility code type for that as well.